src/clr-angular/main.scss

```diff
@@ -9,5 +9,6 @@
 
 // OG Theme: This is for generating the OG Clarity Theme
 
+//@import './utils/theme.dark.clarity';
```

Comment:
... do we need this ?

Reply:
oops! thanks, it's forgotten. This was for hard-wired testing of the component with the dark theme. Removing.